x-pack/plugin/security/src/test/java/org/elasticsearch/xpack/security/rest/action/user/RestGetUserPrivilegesActionTests.java

```diff
@@ -114,11 +117,22 @@ public void testBuildResponse() throws Exception {
             {
               "cluster": [ "monitor", "manage_ml", "manage_watcher" ],
               "global": [
+                {
+                  "application": {
+                  },
+                  "profile": {
+                    "update": {
+                      "applications": [ "app*" ]
+                    }
+                  }
+                },
                 {
                   "application": {
                     "manage": {
                       "applications": [ "app01", "app02" ]
                     }
+                  },
+                  "profile": {
                   }
```

Comment:
This output looks weird. Each entry in `global` has a dummy section (either `application` or `profile`). It is better to output something like:
```
"global": [
  {
    "application": { ... },
    "profile": { ... }
  }
]
```

I am happy for this to be a follow-up PR. 

PS: Strictly speaking, the value of the field "global" being an array instead of an object is also weird since it does not match the synatx used for role descriptor. But I don't think we want to introduce breaking change.

Reply:
I agreed the formatting is odd.
I have pushed 77fcc4726436fde98f0d830bd8098193445a5e3f , which **doesn't** follow your suggested formatting, but which I believe works better if `global` must be an array.
It looks like:
```
"global": [
  {
    "profile": {
      "update": {
        "applications": [ "app*" ]
      }
    }
  },
  {
    "application": {
      "manage": {
        "applications": [ "app01", "app02" ]
      }
    }
  }
]
```

```

Reply:
Please let me know if you dislike it.

Reply:
This looks fine to me. Thanks!